hugo/assets/scss/components/prevnext.scss

```diff
@@ -2,105 +2,87 @@
 @import '../config/sizes';
 @import '../mixins/container';
 @import '../mixins/screen';
-@import '../mixins/separator-line';
 @import '../mixins/typography';
 
 .prevnext {
     $self: &;
 
-    --prevnext-color: #{ $c-blue };
+    --prevnext-bg-color: #{ $c-blue--lighter };
+    --prevnext-color: #{ $c-black };
+    --prevnext-icon-color: #{ $c-blue };
+    --prevnext-icon-size: 30px;
 
-    padding: 1.5rem 0;
+    padding-bottom: 2rem;
     position: relative;
 
     &__container {
         @include container($w-content);
```

Comment:
we might need to change the container width to whatever docs has container width because now the prevnext arrow are not aligned with the rest of the footer. In design they are aligned with the footer  (yellow line)
![Screenshot from 2023-08-25 10-16-57](https://github.com/cue-lang/cuelang.org/assets/122357989/b1401b71-97cf-455d-8499-9297421da9dd)